src/core/poetry.lock

```diff
@@ -539,13 +542,13 @@ files = [
 
 [[package]]
 name = "openai"
-version = "1.45.0"
+version = "1.45.1"
 description = "The official Python library for the openai API"
-optional = false
+optional = true
```

Comment:
@coreyhu Notice how openai was not marked optional previously despite the openai group having `optional = true` in it.